packages/legacy-scripting-runner/test/example-app/api-key-auth.js

```diff
@@ -41,5 +45,8 @@ module.exports = {
       source: beforeRequestSource,
       args: ['request', 'z', 'bundle']
     }
+  ],
+  afterResponse: [
+    { source: afterResponseSource, args: ['response', 'z', 'bundle'] }
```

Comment:
To not default `afterResponse` to `[throwForStatus]` every example app needs to use `legacyScripting.afterResponse` which defaults to a no-op like we will for visual builder apps.